spylib/utils/session_token.py

```diff
@@ -0,0 +1,112 @@
+from __future__ import annotations
+
+from typing import Any, ClassVar, Dict, Optional
+from pydantic.main import BaseModel
+from pydantic.networks import HttpUrl
+from datetime import datetime, timedelta
+
+import jwt
```

Comment:
Why not?
```suggestion
from jwt import decode
```

Reply:
I thought it would be clearer what decode does if I used it directly from `jwt`. Let me know if you have a preferred syntax.